src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -755,6 +758,11 @@ private BidderRequest createBidderRequest(BidderPrivacyResult bidderPrivacyResul
 
         final App bidRequestApp = bidRequest.getApp();
         final Site bidRequestSite = bidRequestApp == null ? bidRequest.getSite() : null;
+
+        if (bidRequestApp != null && bidRequest.getSite() != null) {
+            debugWarnings.add("BidRequest contains app and site. Removed site object");
```

Comment:
Thanks a lot :smiley:  These debug warnings are logged to console, right? How to monitor those in production? Especially the log message doesn't answer for what account this error is produced, right?

Reply:
DebugWarnigs will be placed in BidResponse(they are not about logging), so it can be monitored